poppy/dms.py

```diff
@@ -391,7 +391,7 @@ def _get_surface_via_gaussian_influence_functions(self, wave):
 
         interpolated_surface = np.zeros(wave.shape)
 
-        crosstalk = 0.15  # amount of crosstalk on advancent actuator
+        crosstalk = 0.15  # amount of crosstalk on adjacent actuator
```

Comment:
I'm not a domain expert, if this is a domain term, I'm sorry, and I'd been interested to learn about it. That said, the only Google hit for `advancent actuator` is this specific instance, so I'm guessing it's a typo. What it's a typo of, I'm much less confident.